[DOCS-15207] Document Create with AI for Forms (#38571)
DataDog/documentation: content/en/actions/forms/_index.md

[100]: /bits_ai/bits_chat/

{{% /tab %}}

{{% tab "Blank form" %}}
1. Select {{< ui >}}Start with a blank form{{< /ui >}} and click {{< ui >}}Continue{{< /ui >}}.
1. Name your form and optionally add a description and theme color. Click {{< ui >}}Continue{{< /ui >}}.
1. To add a component, click {{< ui >}}Add Component{{< /ui >}}, or in the {{< ui >}}Fields{{< /ui >}} panel, click the plus **+** icon. See [Form components][3] for the full list of component types and their options.
1. Click {{< ui >}}Publish{{< /ui >}} or {{< ui >}}Publish Changes{{< /ui >}} to make the form available to respondents.

[3]: /actions/forms/components/

{{% /tab %}}

{{% tab "Blueprint" %}}
Blueprints are starter forms for common use cases, pre-loaded with sample questions. Some blueprints include a pre-configured automation. Available blueprints include Developer Experience Survey, IDP Feedback, Case Management Service Request, Report an Incident, Bug Report, On-Call Escalation, Post-Incident Review, and more.

1. Select {{< ui >}}Create from blueprint{{< /ui >}} and browse the available templates.
1. Select a blueprint and click {{< ui >}}Continue{{< /ui >}}.

> 1. Name your form and optionally add a description and theme color. Click {{< ui >}}Continue{{< /ui >}}.

Reviewer: Does this need another step in here to make optional edits to the blueprint questions? Otherwise, people might think that their only choice is to publish the templated form. Could work the form components link in here too, which would have some more context 🙂

Author: ```suggestion
1. Name your form and optionally add a description and theme color. Click {{< ui >}}Continue{{< /ui >}}.
1. To further customize your form, see [Form components][3].
```

1. Click {{< ui >}}Publish{{< /ui >}} or {{< ui >}}Publish Changes{{< /ui >}} to make the form available to respondents.

> To further customize your form, see [Form components][3].

Author: ```suggestion
```

[3]: /actions/forms/components/
{{% /tab %}}

{{% tab "Import" %}}
You can import an existing form from a PDF or JSON file.

1. Select {{< ui >}}Import a form{{< /ui >}}. An import dialog opens.
1. Choose a source and follow the prompts.

> 1. Name your form and optionally add a description and theme color. Click {{< ui >}}Continue{{< /ui >}}.

Reviewer: Same thing here! I think another step to make customizations would make sense

Author: ```suggestion
1. Name your form and optionally add a description and theme color. Click {{< ui >}}Continue{{< /ui >}}.
1. To further customize your form, see [Form components][3].
```

1. Click {{< ui >}}Publish{{< /ui >}} or {{< ui >}}Publish Changes{{< /ui >}} to make the form available to respondents.

> To further customize your form, see [Form components][3].

Author: ```suggestion
```

[3]: /actions/forms/components/
{{% /tab %}}
{{< /tabs >}}

To preview or share your form:

> 1. Click {{< ui >}}View{{< /ui >}} to display the form as it appears to respondents. Click {{< ui >}}Edit{{< /ui >}} to return to the creator view.

Reviewer: I see Preview and Close Preview in the UI. Is that current? Or maybe you're seeing a future release I'm not?

Author: Great catch!

```suggestion
1. Click {{< ui >}}Preview{{< /ui >}} to view the form as it appears to respondents.
```

[3]: /actions/forms/components/
{{% /tab %}}
{{< /tabs >}}

To preview or share your form:
1. Click {{< ui >}}View{{< /ui >}} to display the form as it appears to respondents. Click {{< ui >}}Edit{{< /ui >}} to return to the creator view.
1. Click {{< ui >}}Share{{< /ui >}} to copy the form link or configure sharing options.

## Form settings

From the [Forms][2] page, click a form to open it in the editor. In the editor header, click the gear <i class="icon-cog-2"></i> icon to access the following settings:

| Setting | Description |
|---------|-------------|
| Accepting Responses | Set the form as active or inactive. When inactive, the form does not accept new responses. You can also set an end date to automatically close the form on a specific date. Only available for published forms. |
| Anonymous Responses | When enabled, respondent emails are not stored. |
| Manage Permissions | Configure who can view and edit the form, and who can view submitted responses. See [Manage access](#manage-access). |
| Clone Form | Create a copy of the form. |
| Import Form | Import fields from a PDF or JSON file into the current form. |
| Export Form (JSON) | Download the form as a JSON file. |

> <!-- | Limit to 1 response | When enabled, prevents the same person from submitting more than one response. | -->

Author: ```suggestion
```

## Add a form to a dashboard

To add a form to a dashboard from the form editor:
1. From the [Forms][2] page, click a form to open it in the editor.

> 1. Click {{< ui >}}Share{{< /ui >}}.

Reviewer: I see another step here to click "Share within Datadog"? Should that be in here, or has it been taken out?

Author: Yes!

```suggestion
1. Click the {{< ui >}}Share{{< /ui >}} dropdown and select {{< ui >}}Share within Datadog{{< /ui >}} .
```

[8]: https://app.datadoghq.com/workflow/blueprints
[9]: /actions/workflows/build/#build-a-workflow-with-the-workflow-builder
[10]: https://app.datadoghq.com/workflow

> [11]: /bits_ai/bits_chat/

Author: ```suggestion
```
